server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -1039,7 +1039,10 @@ public void testHistoryRetention() throws Exception {
         assertThat(recoveryState.getTranslog().recoveredOperations(), greaterThan(0));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/105122")
+    @TestLogging(
+        reason = "https://github.com/elastic/elasticsearch/issues/105122",
```

Comment:
I think `TestIssueLogging` is the better annotation in this case (although it's not functionally different from `TestLogging`.

```suggestion
    @TestIssueLogging(
        issueUrl = "https://github.com/elastic/elasticsearch/issues/105122",
```